moped-editor/src/views/projects/projectView/ProjectComponents/TheMap.js

```diff
@@ -0,0 +1,324 @@
+import { useState, useMemo } from "react";
```

Comment:
just want to mention that there was zero thought behind calling this component `TheMap` and i think `Map` would be better 🤪

Reply:
Ok cool - thought is sounded pretty snazzy myself 😄